tests/functional/test_calculator.py

```diff
@@ -53,9 +53,9 @@ def test_failure(self):
         self.assertIn(error_header, output)
 
         feature_stack_frame = """
-  File "features/wrong_expectations.feature", line 11, in Add two numbers
+  File "{root}/tests/simple_app/features/wrong_expectations.feature", line 11, in Add two numbers
     Then the result should be 40 on the screen
-        """.strip()
+        """.strip().format(root=dirname(dirname(dirname(__file__))))  # noqa
```

Comment:
It's changing because I'm not relpath'ing something when generating the class. It's more in line with Python which doesn't do that either.